tsconfig.json

```diff
@@ -7,7 +7,9 @@
     "outDir": "dist",
     "types": ["jest"],
     "esModuleInterop": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "moduleResolution": "node",
```

Comment:
What was this for?

Reply:
I changed the target to  ES2020, which required this additional property to continue working.